### PR TITLE
More extensive robots.txt docs

### PR DESCRIPTION
**Pull request by 8ctopus**
Added documentation on how to block search engines indexation of gitea repository

### CODE REVIEW

File: docs/content/doc/advanced/customizing-gitea.en-us.md [hunk 1]
```diff
@@ -143,9 +137,9 @@ To add a custom license, add a file with the license text to `custom/options/lic
 
 ### Locales
 
-Locales are managed via our [crowdin](https://crowdin.com/project/gitea).  
-You can override a locale by placing an altered locale file in `custom/options/locale`.  
-Gitea's default locale files can be found in  the [`options/locale`](https://github.com/go-gitea/gitea/tree/master/options/locale) source folder and these should be used as examples for your changes.  
+Locales are managed via our [crowdin](https://crowdin.com/project/gitea).
+You can override a locale by placing an altered locale file in `custom/options/locale`.
+Gitea's default locale files can be found in  the [`options/locale`](https://github.com/go-gitea/gitea/tree/master/options/locale) source folder and these should be used as examples for your changes.
```
Comment (bagasme) [inline docs/content/doc/advanced/customizing-gitea.en-us.md]: Whitespace deletion (unrelated changes) here?

Comment (6543) [reply docs/content/doc/advanced/customizing-gitea.en-us.md]: would say this is ok ... code formation in seperate PR is just noice ...

**Review by 6543 — APPROVED** [review]

**Comment by zeripath** [comment]
To be honest I think it would be reasonable to just add a very restricted default robots.txt and get users to override that if they want things to be indexed.

**Comment by 6543** [comment]
worth a look is https://codeberg.org/robots.txt ...

**Comment by bagasme** [comment]
> worth a look is https://codeberg.org/robots.txt ...

`ia_archiver` bot is from [Wayback Machine](web.archive.org), used to snapshot a site for archival.

File: docs/content/doc/advanced/customizing-gitea.en-us.md (hunk 1 above)
Comment (8ctopus) [reply docs/content/doc/advanced/customizing-gitea.en-us.md]: @bagasme my text editor is configured to automatically trim end of line spaces.

**Review by 8ctopus — COMMENTED** [review]
@zeripath to have a restrictive robots.txt was already suggested before but appears not to have reached consensus. #705

File: docs/content/doc/advanced/search-engines-indexation.en-us.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+date: "2019-12-31T13:55:00+05:00"
+title: "Avanced: Search Engines Indexation"
```
Comment (guillep2k) [inline docs/content/doc/advanced/search-engines-indexation.en-us.md]:
```suggestion
title: "Advanced: Search Engines Indexation"
```

**Review by guillep2k — APPROVED** [review]
Just a typo. Otherwise LGTM

**Comment by 8ctopus** [comment]
@guillep2k thanks, what's LGTM?

File: docs/content/doc/advanced/customizing-gitea.en-us.md [hunk 3]
```diff
@@ -172,5 +166,5 @@ A full list of supported emoji's is at [emoji list](https://gitea.com/gitea/gite
 
 ## Customizing the look of Gitea
 
-As of version 1.6.0 Gitea has built-in themes. The two built-in themes are, the default theme `gitea`, and a dark theme `arc-green`. To change the look of your Gitea install change the value of `DEFAULT_THEME` in the [ui](https://docs.gitea.io/en-us/config-cheat-sheet/#ui-ui) section of `app.ini` to another one of the available options.  
+As of version 1.6.0 Gitea has built-in themes. The two built-in themes are, the default theme `gitea`, and a dark theme `arc-green`. To change the look of your Gitea install change the value of `DEFAULT_THEME` in the [ui](https://docs.gitea.io/en-us/config-cheat-sheet/#ui-ui) section of `app.ini` to another one of the available options.
```
Comment (6543) [inline docs/content/doc/advanced/customizing-gitea.en-us.md]:
```suggestion
As of version 1.6.0 Gitea has built-in themes. The two built-in themes are, the default theme `gitea`, and a dark theme `arc-green`. To change the look of your Gitea install change the value of `DEFAULT_THEME` in the [ui](https://docs.gitea.io/en-us/config-cheat-sheet/#ui-ui) section of `app.ini` to another one of the available options.  
```

File: docs/content/doc/advanced/customizing-gitea.en-us.md [hunk 4]
```diff
@@ -143,9 +137,9 @@ To add a custom license, add a file with the license text to `custom/options/lic
 
 ### Locales
 
-Locales are managed via our [crowdin](https://crowdin.com/project/gitea).  
-You can override a locale by placing an altered locale file in `custom/options/locale`.  
-Gitea's default locale files can be found in  the [`options/locale`](https://github.com/go-gitea/gitea/tree/master/options/locale) source folder and these should be used as examples for your changes.  
+Locales are managed via our [crowdin](https://crowdin.com/project/gitea).
```
Comment (6543) [inline docs/content/doc/advanced/customizing-gitea.en-us.md]:
```suggestion
Locales are managed via our [crowdin](https://crowdin.com/project/gitea).  
```

File: docs/content/doc/advanced/customizing-gitea.en-us.md [hunk 5]
```diff
@@ -143,9 +137,9 @@ To add a custom license, add a file with the license text to `custom/options/lic
...
+You can override a locale by placing an altered locale file in `custom/options/locale`.
```
Comment (6543) [inline docs/content/doc/advanced/customizing-gitea.en-us.md]:
```suggestion
You can override a locale by placing an altered locale file in `custom/options/locale`.  
```

File: docs/content/doc/advanced/customizing-gitea.en-us.md (hunk 1 above)
Comment (6543) [inline docs/content/doc/advanced/customizing-gitea.en-us.md]:
```suggestion
Gitea's default locale files can be found in  the [`options/locale`](https://github.com/go-gitea/gitea/tree/master/options/locale) source folder and these should be used as examples for your changes.  
```

Comment (6543) [reply docs/content/doc/advanced/customizing-gitea.en-us.md]: @bagasme, @8ctopus  ... found out `  ` is newline on markdown ... so would be nice if they stay

File: docs/content/doc/advanced/customizing-gitea.en-us.md [hunk 6]
```diff
@@ -134,7 +128,7 @@ To add custom .gitignore, add a file with existing [.gitignore rules](https://gi
 
 ### Labels
 
-To add a custom label set, add a file that follows the [label format](https://github.com/go-gitea/gitea/blob/master/options/label/Default) to `custom/options/label`  
+To add a custom label set, add a file that follows the [label format](https://github.com/go-gitea/gitea/blob/master/options/label/Default) to `custom/options/label`
```
Comment (6543) [inline docs/content/doc/advanced/customizing-gitea.en-us.md]:
```suggestion
To add a custom label set, add a file that follows the [label format](https://github.com/go-gitea/gitea/blob/master/options/label/Default) to `custom/options/label`  
```

**Review by 6543 — CHANGES_REQUESTED** [review]
`  ` on the end of line in makdown is a /newline
@8ctopus jsut select my suggestions and commit it in one batch :)

PS: just lerned :)

**Comment by 8ctopus** [comment]
@6543 ooops!, sorry about that, I didn't know it either. I'll fix it and commit again.

**Review by 6543 — APPROVED** [review]
Thanks

**Comment by 8ctopus** [comment]
@6543 commited in one batch (didn't know how to do it from github so I did it on my computer).

File: docs/content/doc/advanced/customizing-gitea.en-us.md (hunk 1 above)
Comment (8ctopus) [reply docs/content/doc/advanced/customizing-gitea.en-us.md]: @bagasme turns out you had a good point. end of line whitespace has its use with markdown which I didn't know about.

**Comment by 6543** [comment]
You can do it on the "Files changed" view ... a nice way if you are not at home (and change PR with your webbrowser ...)

In the end it doesnt mather how you do it :)

**Comment by guillep2k** [comment]
> 
> 
> @guillep2k thanks, what's LGTM?

Means "looks good to me". 😄
